x-pack/plugins/cloud_security_posture/common/schemas/benchmark.ts

```diff
@@ -8,7 +8,7 @@ import { type TypeOf, schema } from '@kbn/config-schema';
 
 export const DEFAULT_BENCHMARKS_PER_PAGE = 20;
 export const BENCHMARK_PACKAGE_POLICY_PREFIX = 'package_policy.';
-export const benchmarksInputSchema = schema.object({
+export const getBenchmarksQueryParamsSchema = schema.object({
```

Comment:
the naming structure is the API: GET /benchamrks
The part of the API, query params
And the type of this object: Schema

would you advice to do it otherwise?
This way it is easy to find anything related to get benchmarks api

Reply:
`get` is a common prefix for a getter function. random examples from our codebase: `getFindingsQuery`, `getSideNavItems`, `getTopRisks`

to me `getBenchmarksQueryParamsSchema` reads like a getter function but it's actually a schema for the `GET` http verb. not that critical, but if we're considering a pattern here, i'd go with `benchmarksQueryParamsSchemaGet` 

discovery will be the same but no possible dual meaning (to me at least 😄 )

Reply:
how about `benchmarksGetQueryParamsSchema`?

Reply:
i'm good with either, not that critical. to me the argument still stands though, regardless of others.